### ISSUE TITLE
Fluxstore Pro-1.4.3

### ISSUE THREAD

**Issue by anandnathtiwari** [inspireui/support#1011]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets 
> - After creating an issue, please copy the URL to the Product link on http://codecanyon.net - to help us know you have purchased our product (example: https://tppr.me/DJA8L)
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue

I have couple of issue in version 1.4.3. Please guide me. I am using Opencart 3.0.3.2 
(1) Payment methods not show. in website I use CCAvenue as payment Gateway 
(2) in Product list some product not show image 
(3) in Recent product list—products repeated itself many times and products image size is not equal.

Thanks Anand Nath tiwari

2
[config.zip](https://github.com/inspireui/support/files/3562039/config.zip)
. Config files attached.

**Comment by minhcasi** [comment]
Hi Anand, thank you for your feedback, could we know your purchased screenshot?

**Comment by anandnathtiwari** [comment]
![images Missing in recent collection](https://user-images.githubusercontent.com/17059651/64072343-7c188880-ccaa-11e9-9074-3c24b8105f0e.jpg)
![Images repeated and not alignment](https://user-images.githubusercontent.com/17059651/64072344-7c188880-ccaa-11e9-9334-20df25945267.jpg)
![payment Issue](https://user-images.githubusercontent.com/17059651/64072345-7cb11f00-ccaa-11e9-8a12-32316d69a159.jpg)

**Comment by anandnathtiwari** [comment]
![website checkout page](https://user-images.githubusercontent.com/17059651/64072410-56d84a00-ccab-11e9-9ad0-8c4118667fc0.jpg)

**Comment by anandnathtiwari** [comment]
In website Payment method show, but in app it gives error. Also Taxes are not show in App.

Thanks
Anand Nath Tiwari

**Comment by anandnathtiwari** [comment]
Any update on my issue ?

**Comment by minhcasi** [comment]
@anandnathtiwari - sorry for the late response as we are one Labor holiday, the team will back on tomorrow for checking your issue - thanks

@minhtruong315

**Comment by minhtruong315** [comment]
@anandnathtiwari I can not login to check the payment method. it require account must be approved before login.

**Comment by minhtruong315** [comment]
@anandnathtiwari 
(2) in Product list some product not show image
(3) in Recent product list—products repeated itself many times and products image size is not equal.
I checked your site, it don't response images list. you can check products api on your website source code. it's in file
catalog/controller/extension/mstore/product.php

**Comment by anandnathtiwari** [comment]
please login  email:  api@test.com and pass 123456

Thanks

**Comment by anandnathtiwari** [comment]
I also approved all your customers

**Comment by minhtruong315** [comment]
@anandnathtiwari to fix payment method, you need to update EnableShipping:true in config.dart.
edit fromOpencartJson contructor in models/shipping_method.dart
```
ShippingMethod.fromOpencartJson(Map<String, dynamic> parsedJson) {
    Map<String, dynamic> quote = parsedJson["quote"];
    Map<String, dynamic> item = quote.values.isNotEmpty ? quote.values.toList()[0] : null;
    id =  item != null ? item["code"] : "0";
    title = parsedJson["title"] ?? id;
    description = item != null && item["title"] != null ? item["title"] : "";
  }
```

**Comment by anandnathtiwari** [comment]
Previous issue resolved. But Please see attached file and guide me. order booked without payment.

Tax is not show in checkout page.

![Order-booked-without-payment](https://user-images.githubusercontent.com/17059651/64229052-52f43400-cf06-11e9-8f42-713d08f6a894.jpg)
![payment Issue](https://user-images.githubusercontent.com/17059651/64229053-52f43400-cf06-11e9-90af-2557a63530ae.jpg)

**Comment by anandnathtiwari** [comment]
![ccavenue-image-not-show](https://user-images.githubusercontent.com/17059651/64229137-8fc02b00-cf06-11e9-90d9-4ef387643f38.jpg)

**Comment by minhtruong315** [comment]
@anandnathtiwari you must choose shipping method before get payment methods.

**Comment by minhtruong315** [comment]
@anandnathtiwari if you want to show image for payment method. you can update Payments in config.dart
```
const Payments = {
  "paypal": "assets/icons/payment/paypal.png",
  "stripe": "assets/icons/payment/stripe.png",
  "razorpay": "assets/icons/payment/razorpay.png",
};
```

**Comment by anandnathtiwari** [comment]
I already add images in config.dart. if you check my previously uploaded config.dart file, you find it. but it not show

**Comment by minhtruong315** [comment]
you put wrong payment code. pls try this
"ccavenuepay":"assets/icons/payment/ccavenue.png",

**Comment by anandnathtiwari** [comment]
Thanks. Payment Image show.
